runtime-image/test/test.ts

```diff
@@ -1,21 +1,32 @@
 
 import * as assert from 'assert';
-import {spawn} from 'child_process';
-import {exec} from 'child_process';
-import * as uuid from 'node-uuid';
+import * as Docker from 'dockerode';
+import * as path from 'path';
 import * as request from 'request';
-import * as util from 'util';
+
+const tar: {pack: (dir: string) => any} = require('tar-fs');
+
+const RUN_TIMEOUT = 3000;
+const BUILD_TIMEOUT = 5 * 60 * 1000;
+const PORT = 8080;
+
+let host = 'localhost';
```

Comment:
I'll combine these into one expression and use `const`.

```typescript
const host = process.env.DOCKER_HOST ?
    process.env.DOCKER_HOST.split('//')[1].split(':')[0] :
    'localhost';
```

Reply:
Done